### PR TITLE
Add CoW AMM Deployer tutorial

### PR DESCRIPTION
**Pull request by yvesfracari**
# Description

Add a tutorial about the to-be-launched CoW AMM Deployer Safe App.

### CODE REVIEW

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 6
+---
+
+# CoW AMM Deployer
+The [CoW AMM Deployer](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fdeploy-cow-amm.bleu.fi&chain=et) is a user-friendly Safe App that simplifies the process of deploying and managing [CoW AMMs](https://cow.fi/cow-amm) (Automated Market Makers) from Safe wallets. It enables users without programming skills to take advantage of the benefits of CoW AMM. 
+
+The app was developed by [bleu](https://github.com/bleu-fi) with sponsorship from the [CoW Grants DAO](https://grants.cow.fi/).
+
+This app is only accessible from a Safe wallet. **It is recommended that the Safe wallet only be used for the purpose of deploying the CoW AMM.** Also please take note that the Safe wallet must already have the tokens you plan to use to provide liquidity to the AMM before creating it.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
This app is only accessible from a Safe wallet. 

:::caution

The Safe wallet you use to create a CoW AMM should **ONLY** be used for the purpose of deploying a CoW AMM.

:::

:::tip

Provide the tokens for the CoW AMM to your Safe wallet **before** creating the CoW AMM. Beware **any** imbalance between tokens will _automatically_ be traded.

:::
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Prerequisites
+Before using the CoW AMM Deployer, ensure that you have the following:
+1. A Safe wallet: If you don't have one, follow these steps to create a new Safe wallet:
+   - Go to the [Safe website](https://app.safe.global/welcome) and click on "Create new Safe".
+   - Choose the network (e.g., Ethereum) and the number of required confirmations.
+   - Add the owners of the Safe wallet and set the threshold for confirmations.
+   - Review and create the Safe wallet.
+2. Necessary tokens: Transfer both tokens that you want to provide liquidity for to your Safe wallet. Make sure you have sufficient amounts of each token.
+
+Once that's done, you're ready to create your first AMM!
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
Once that's done, you're ready to create your first CoW AMM!
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Creating an AMM
+To create a new CoW AMM using the deployer, follow these steps:
+1. Access the CoW AMM Deployer app from your Safe wallet.
+2. On the homepage, click on the "Create AMM" button.
+3. Fill in the required information in the AMM creation form:
+   - **Token Pair**: Select the token pair you want to provide liquidity for. Both tokens should already be in your Safe wallet.
+   - **Price Oracle**: Choose the price oracle (e.g., Balancer, UniSwap V3) to provide price hints to the AMM. Enter the pool identifier used by the selected protocol. You can also load this information from the protocol subgraph using the provided button. If the button fails, it means there isn't a pool with at least $1000 USD of TVL for that token pair.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
   - **Price Oracle**: Choose the price oracle (e.g., Balancer, UniSwap V3) to provide price hints to the watch-tower about your CoW AMM. Enter the pool identifier used by the selected protocol. You can also load this information from the protocol subgraph using the provided button. If the button fails, it means there isn't a pool with at least $1000 USD of TVL for that token pair.
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+   - **Minimum Trade Amount**: Specify the minimum amount of the first token required for each trade. For most tokens, this value is automatically filled. If not, a good strategy is to calculate the equivalent of $10 USD in the first token.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
   - **Minimum Trade Amount**: Specify the minimum amount of the first token required for each trade. For most tokens, this value is automatically filled. If not, a strategy may be to calculate the equivalent of $10 USD in the first token.
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+   - **Fallback Handler**: If your wallet is unsupported, upgrade to a Safe with the special fallback handler. [Learn all about the new Safe fallback handler](https://blog.cow.fi/all-you-need-to-know-about-cow-swaps-new-safe-fallback-handler-8ef0439925d1) and its relevance to Safe in this [overview](https://help.safe.global/en/articles/40838-what-is-a-fallback-handler-and-how-does-it-relate-to-safe). Naturally, the upgrade process has undergone [extensive auditing](https://github.com/cowprotocol/composable-cow/tree/ab3addad9bc05acdbf5fb040eb5c336209b58e31/audits) for increased peace of mind.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
   - **Fallback Handler**: If your wallet is unsupported, upgrade to a Safe with the special fallback handler. [Learn all about the new Safe Extensible Fallback Handler](https://blog.cow.fi/all-you-need-to-know-about-cow-swaps-new-safe-fallback-handler-8ef0439925d1) and its relevance to Safe in this [overview](https://help.safe.global/en/articles/40838-what-is-a-fallback-handler-and-how-does-it-relate-to-safe). Naturally, the upgrade process has undergone [extensive auditing](https://github.com/cowprotocol/composable-cow/tree/ab3addad9bc05acdbf5fb040eb5c336209b58e31/audits) for increased peace of mind.
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+The "Manager" page provides an overview of your CoW AMM LP position and allows you to manage and edit its parameters.
+
+The page displays the following information:
+- **Total Value**: The total value of the liquidity provided to the AMM, calculated based on the current token prices.
+- **Token Amounts**: The individual amounts of each token in the AMM liquidity pool.
+- **Price Oracle**: The selected price oracle used by the AMM.
+- **Swaps Link**: A link to view the swaps executed by the CoW AMM on the CoW Explorer.
+
+From the "Manager" page, you can perform two actions:
+1. **Stop AMM LP Position**: Clicking this button will redirect you to a confirmation page. After confirming, the AMM will be stopped, and the tokens will remain in your Safe wallet.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
1. **Stop AMM LP Position**: Clicking this button will redirect you to a confirmation page. After confirming, the CoW AMM will be stopped, and the tokens in your Safe wallet will no longer be used to provide liquidity.
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+## Additional Resources
+- [CoW AMM Documentation](https://cow.fi/cow-amm)
+- [CoW Grants DAO](https://grants.cow.fi/)
+- [Safe Wallet Documentation](https://help.safe.global/)
+- [CoW Swap (Safe App)](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fswap.cow.fi&chain=eth)
+
+## Troubleshooting and FAQs
+
+### Frequently Asked Questions
+1. **Can I create multiple AMMs with the same Safe wallet?**
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
1. **Can I create multiple CoW AMMs with the same Safe wallet?**
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+   - No.
+
+2. **What happens to my tokens when I stop an AMM LP position?**
+   - When you stop an AMM LP position, the tokens that were provided as liquidity will be returned to your Safe wallet. You can then withdraw or use them for other purposes.
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
   - When you stop an AMM LP position, your tokens in your Safe will no longer be used to provide liqudity. As the CoW AMM **IS** the Safe when there is an active CoW AMM, the liquidity never leaves your Safe wallet! You can then withdraw or use them for other purposes.
```

**Review by mfw78 — CHANGES_REQUESTED** [review]
Looks great overall, find some feedback in-line suggestions

**Comment by yvesfracari** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+Once that's done, you're ready to create your first CoW AMM!
+
+## Creating an AMM
+To create a new CoW AMM using the deployer, follow these steps:
+1. Access the CoW AMM Deployer app from your Safe wallet.
+2. On the homepage, click on the "Create AMM" button.
+3. Fill in the required information in the AMM creation form:
+   - **Token Pair**: Select the token pair you want to provide liquidity for. Both tokens should already be in your Safe wallet.
+   - **Price Oracle**: Choose the price oracle (e.g., Balancer, UniSwap V3) to provide price hints to the watch-tower about your CoW AMM. Enter the pool identifier used by the selected protocol. You can also load this information from the protocol subgraph using the provided button. If the button fails, it means there isn't a pool with at least $1000 USD of TVL for that token pair.
```
Comment (mendesfabio) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx]:
```suggestion
   - **Price Oracle**: Choose the price oracle (e.g., Balancer, Uniswap V2) to provide price hints to the watch-tower about your CoW AMM. Enter the pool identifier used by the selected protocol. You can also load this information from the protocol subgraph using the provided button. If the button fails, it means there isn't a pool with at least $1000 USD of TVL for that token pair.
```

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+1. **Can I create multiple CoW AMMs with the same Safe wallet?**
+   - No.
+
+2. **What happens to my tokens when I stop an AMM LP position?**
+   - When you stop an AMM LP position, your tokens in your Safe will no longer be used to provide liqudity. As the CoW AMM **IS** the Safe when there is an active CoW AMM, the liquidity never leaves your Safe wallet! You can then withdraw or use them for other purposes.
+
+3. **Can I change the token pair of an existing AMM?**
+   - No, once an AMM is created with a specific token pair, you cannot change the token pair. If you want to provide liquidity for a different token pair, you need to create a new AMM.
+
+For further assistance or questions, join the [CoW Protocol Discord](https://discord.com/invite/cowprotocol) and ask in the `#tech-talk` channel.
```
Comment (mendesfabio) [inline docs/cow-protocol/tutorials/cow-amm-deployer.mdx:83]: I wonder if we should redirect to #tech-talk or #cow-amm channel

**Review by mendesfabio — APPROVED** [review]

File: docs/cow-protocol/tutorials/cow-amm-deployer.mdx (hunk 10 above)
Comment (mfw78) [reply docs/cow-protocol/tutorials/cow-amm-deployer.mdx:83]: From my understanding it's not possible (I don't think?) to set the invite link for discord to specific channels?

**Review by mfw78 — APPROVED** [review]

**Pull request merged**